app/controllers/admin/settings_controller.rb

```diff
@@ -1,39 +1,31 @@
 class Admin::SettingsController < Admin::BaseController
-  before_action :find_admin_setting
+  before_action :load_admin_setting
 
   def index
     authorize @admin_setting
   end
 
   # PUT /admin_settings/1
-  # PUT /admin_settings/1.xml
   def update
     authorize @admin_setting
-    authorized_params = policy(@admin_setting).verify_permitted_params(admin_setting_params) 
 
-    if authorized_params == true && @admin_setting.update(admin_setting_params)
-      flash[:notice] = ts("Archive settings were successfully updated.")
+    if policy(@admin_setting).unpermitted_attributes(params).present?
```

Comment:
`unpermitted_attributes` exists only so we can have this error message. Since we keep restricted settings disabled, the only time admins can easily update settings they don't have permissions to, is when they have the settings form open, but their roles have been changed in the database.

I'm not attached to this error message. I can remove it if we think it's OK to silently ignore settings not allowed by the current admin's roles.

Reply:
Yeah, let's get rid of it.